src/app/components/Button/Button.tsx

```diff
@@ -6,16 +6,12 @@ import Typography from '../Typography/Typography';
 type ButtonProps = {
   children: ReactNode;
   className?: string;
-  onButtonClick: () => void;
+  onClick?: () => void;
```

Comment:
I wouldn't have this optional, it should always be defined, even in a story in storybook. Just like your `handleSubmit` in your `SearchBar` component

Reply:
Could you please check if this is the correct solution.